features/items.md

```diff
@@ -0,0 +1,237 @@
+
+    VALUE\ WITH\ SPACES=value with spaces
+
+## Icons
+OpenHAB provides you a set of basic icons by default. However if you wish to use custom icons you need to create the folder ```${openhab_home}/conf/icons/classic/``` and place all your icons inside. These icons will be used in all of the openHAB frontends. The images must be in .png or .svg format, and have a name with only small letters and a hyphen or underscore (if required). The PaperUI interface (or via the classicui.cfg or basicui.cfg files) allows you to define whether you use Vector (.svg) or Bitmap (.png) icon files.
+
+As an example, to use a custom icon called `switch-off.svg` the correct syntax is `<switch-off>`.  
+
+## Dynamic icons
+You can dynamically change the icon depending on the item state. You have to provide a default file and one icon file per state with the states name append to the icons name.
+
+Example:
+
+    switch.svg
+    switch-off.svg
+    switch-on.svg
+
+If you want to use the dynamically items just use the image name without the added states.
+
+```xtend
+Switch  Light_FrontDoor  "Front Door light is [MAP(en.map):%s]"  <switch>  {somebinding:someconfig}
+```
+
+## Binding config
+The binding configuration is the most import part of an item. It defines from where the item gets it values, and where a given value/command should be sent.
+
+You bind an item to a binding by adding a binding definition in curly brackets at the end of the item definition
+
+    { ns="bindingconfig" }
+
+Where _ns_ is the namespace for a certain binding like "knx", "bluetooth", "serial" etc. Every binding defines what values must be given in the binding configuration string. That can be the id of a sensor, an ip or mac address or anything else. You must have a look at your [[Bindings]] configuration section to know what to use. Some typical examples are:
+
+```java
+Switch  Light_Floor        "Light at Floor"                { knx="1/0/15+0/0/15" }
+Switch  Presence           "I'm at home"                   { bluetooth="123456ABCD" }
+Switch  Doorbell           "Doorbell"                      { serial="/dev/usb/ttyUSB0" }
+Contact Garage             "Garage is [MAP(en.map):%s]"    { zwave="21:command=sensor_binary,respond_to_basic=true" }
+String  Error_Ventilation  "Error in Ventilation %s"       { comfoair="error_message" }
+Number  DiningRoomTemp     "Maximum Away Temp. [%.1f °F]"  { nest="<[thermostats(Dining Room).away_temperature_high_f]" }
+```
+
+When you install a binding through PaperUI it will automatically create a .cfg file in ```${openhab_home}/conf/services/``` for the appropriate binding. Inside these files are a predefined set of variables which are required for the binding to operate. In many cases you will need to view and edit these to suit your system. These variables can hold ip addresses, API keys, user names, passwords etc. These are all in plain text, so be careful who you share these with if some data is sensitive.
```

Comment:
OK, perhaps we should keep this here until the binding docs page is written and we can review any duplication of information on this page?